### ISSUE TITLE
Worse accuracy for MobileNetV3

### ISSUE THREAD

**Issue by NoahLuffy** [megvii-model/FunnelAct#4]
I reimplemented FReLU in PyTorch, and apply it on MobileNetV3. The ImageNet accuracy is as follow:

|model|top1|
|-|-|
|mobilenetv3+hswish|75.2%|
|mobilenetv3+frelu|74.8%|

The code:
```python
class FReLU(nn.Module):
    def __init__(self, in_channels, inplace: bool = False):
        super(FReLU, self).__init__()
        self.inplace = inplace
        self.conv_frelu = nn.Conv2d(in_channels, in_channels, 3, 1, 1, groups=in_channels, bias=False)
        self.bn_frelu = nn.BatchNorm2d(in_channels)

    def forward(self, x):
        x1 = self.conv_frelu(x)
        x1 = self.bn_frelu(x1)
        x = torch.max(x, x1)
        return x
```

Am I missing something important?

**Comment by nmaac** [comment]
We use gaussian initialization with std=0.01. I simply replace relu with frelu and it shows a slight improvement (0.1~0.3). We note that MobileNetV3 is a NAS-searched optimal CNN architecture, once you change the architecture (frelu has an additional dw-conv), you might need to search again on this new architecture to achieve the optimal result.

**Comment by NoahLuffy** [comment]
Thanks for your quick reply. I'll try your suggestions.
